playbooks/roles/ocp-scheduler/README.md

```diff
@@ -0,0 +1,82 @@
+ocp-scheduler
+============
+
+Scheduler profiles are an alternative to configuring a scheduler policy for scheduling pods on cluster nodes. It provides three types of scheduling profiles to configure such as `HighNodeUtilization`, `LowNodeUtilization` and `NoScoring` for efficient node resource usage. 
+This role allows to configure the scheduling profile and running e2e test suite from `openshift-tests-private` repository.
+
+
+Requirements
+------------
+
+- Access to the cluster as a user with the cluster-admin role.
+- The cluster is in a known good state, without any errors.
+
+
+Role Variables
+--------------
+
+| Variable                       | Required | Default                                                 | Comments                                                                           |
+|--------------------------------|----------|---------------------------------------------------------|------------------------------------------------------------------------------------|
+| scheduler_role_enable          | no       | false                                                   | Set the flag `true` to run this playbook.                                          |
+| configure_scheduler_profile    | no       | false                                                   | Set the flag `true` to configure given scheduling profile.                         |
```

Comment:
There is no default value set in the code. So this becomes Required. I feel condition will fail with None value if variable is not defined.

Reply:
Added in the these role variables to `roles/ocp-scheduler/defaults/main.yml` .